Extensions/3D/JsExtension.js

```diff
@@ -804,12 +804,12 @@ module.exports = {
       object
         .addScopedAction(
           'SetCrossfadeDuration',
-          _('Set crossfade duration'),
+          _('Set crossfade duration (in seconds)'),
```

Comment:
I think it's better if you put the unit in the parameter label, rather than on the action label